homeassistant/components/plugwise/number.py

```diff
@@ -96,25 +91,14 @@ def __init__(
         self._attr_unique_id = f"{device_id}-{description.key}"
```

Comment:
`self.actuator = self.device[description.key]` can be removed from line 89

Reply:
Check